app/models/manageiq/providers/ibm_cloud/inventory/parser/power_virtual_servers.rb

```diff
@@ -1,4 +1,5 @@
 class ManageIQ::Providers::IbmCloud::Inventory::Parser::PowerVirtualServers < ManageIQ::Providers::IbmCloud::Inventory::Parser
+  require "sshkey"
```

Comment:
@agrare Opinions on the location of the require?  I don't think it should be top-level because it would get required even if a customer never has an PowerVS provider.  I'm thinking either the sshkeys method itself or initialize.

Reply:
:+1: yeah can move it into the parser method right above https://github.com/ManageIQ/manageiq-providers-ibm_cloud/pull/349/files#diff-f985d1bd514a044797c4062d1d8d94ff3da33e97f6bfa8d182955661184e7d45R252

Reply:
so anti c, anti pythonic (even though I have done includes to some specific functions). anyways doing it.

Reply:
@Kuldip-Nanda yeah, requires in Ruby are very different than in C and Python.  They are really just a normal method call that loads some code at that moment.  As such, we don't want it loaded at boot because it incurs a memory cost that may not be needed, and we'd prefer that were delayed until the last moment.